reference_docs/YAML_models/ietf_v2_document_attributes.yaml

```diff
@@ -0,0 +1,500 @@
+    value_spec: comma-list
+
+  updates:
+    tags: ["Document info"]
+    title: "Updates"
+    description: "A comma-separated list of RFC numbers or Internet-Draft names that this document updates. RFC XML: `rfc@updates`."
+    value_spec: comma-list
+
+  submission-type:
+    tags: ["Document info"]
+    title: "Submission type"
+    description: "Document stream of document described in https://tools.ietf.org/html/rfc7841[RFC7841]. RFC XML: `rfc@submissionType`."
+    value_spec: "enum: IETF, independent, IAB, IRTF"
+    value_default: IETF
+
+  revdate:
+    tags: ["Document info"]
+    title: "Revision date"
+    description: "Latest revision date of document. Default value is current time. Accepts ISO 8601 date. Also accepts `YYYY` year, and `YYYY[-]MM` year/month. For consistency with AsciiDoc, `:revdate:` is given as an ISO 8601 date; the converter breaks it down into day, month name and year RFC XML: `front/date@day`, `front/date@month`, `front/date@year`."
+    value_default: "{current time}"
+
+  area:
+    tags: ["Document info"]
+    title: "Area of document"
+    description: 'Comma delimited text on which IETF area this document relates to. Value should be either the full name or the abbreviation of one of the http://www.ietf.org/iesg/area.html[IETF areas]. RFC XML: `front/area`.'
+
+  workgroup:
+    tags: ["Document info"]
+    title: "Workgroup"
+    description: "Comma delimited text on which IETF or IRTF workgroup or research group this document originates from. RFC XML: `front/workgroup`."
+    more_information: See https://tools.ietf.org/html/rfc7991#section-2.65[RFC 7991]
+
+  keyword:
+    tags: ["Document info"]
+    title: "Keyword"
+    description: "Comma delimited text for singular keywords used for RFC index and metadata. RFC XML: `front/keyword`."
+
+  xml-lang:
+    tags: ["Document info"]
+    title: "XML language"
+    description: "Set the document language. By default this is `en`. RFC XML: `rfc@xml:lang`."
+
+  consensus:
+    tags: ["Document info"]
+    title: "Consensus"
+    description: |
+      Set document consensus for this document. The following values are allowed:
+
+      * `false` (mapped in v2 to: `no`)
+      * `true` (mapped in v2 to: `yes`)
+
+      RFC XML: `rfc@consensus`.
+    value_spec: boolean
+
+#--- Author attributes ---#
+  fullname:
+    tags: ["Author attributes"]
+    title: "Fullname of author"
+    description: "Author's full name. Can set here instead of document header's '`Author`' line. RFC XML: `front/author@fullname`."
+    declaration: ":fullname:, :fullname{_i}:"
+
+  forename_initials:
+    tags: ["Author attributes"]
+    title: "Forename initials"
+    description: |
+      Author's initials excluding surname. Defaults to dynamically calculated initials. Distinct from the AsciiDoc `:initials:` attribute, which includes surname. +
+      RFC XML: `front/author@initials`.
+    declaration: ":forename_initials:, :forename_initials{_i}:"
+
+  lastname:
```

Comment:
@opoudjis @manuel489 
This attribute is a duplicate because in Standoc there is `surname` to capture the `lastname`. 
Is there a functional difference for between `surname` and `lastname`?